torchgeo/trainers/utils.py

```diff
@@ -11,6 +11,24 @@
 import torch.nn as nn
 from torch import Tensor
 from torch.nn.modules import Conv2d, Module
+from torchvision.models.detection.transform import GeneralizedRCNNTransform
+
+
+class GeneralizedRCNNTransformNoOp(GeneralizedRCNNTransform):  # type: ignore[misc]
```

Comment:
What is the type ignore here? Should I add type hints to torchvision?

Reply:
this is the error I get

`torchgeo/trainers/utils.py:17: error: Class cannot subclass
"GeneralizedRCNNTransform" (has type "Any")  [misc]`

Reply:
This doesn't make much sense because the code itself inherits from torch.nn.Module [here](https://github.com/pytorch/vision/blob/966da7e46f65d6d49df3e31214470a4fe5cc8e66/torchvision/models/detection/transform.py#L86)

Reply:
Ahh okay yes likely torchvision doesn't have type hints for this

Reply:
Strange, I see type hints for basically every class method and attribute, not sure where it's getting `Any` from. Oh well.